lib/tasks/webpacker.rake

```diff
@@ -23,4 +23,9 @@ namespace :webpacker do
   task :npm_install do
     system 'npm install'
   end
+
+  desc "Invoke this task if you are sure you don't want to install npm dependencies"
```

Comment:
Description is not accurate however. It should describe what the task instead of _when_ to use it.

_Invoke this task to prevent webpacker from running `npm install`_

or

_Prevents webpacker from running `npm install`_

Reply:
It's not actually just "npm dependencies", `install` can do many things (which is our case).

Reply:
What the differences? what are those many things?

Reply:
_Prevents webpacker from running `npm install`_ would be my choice, however, what about not creating it at all and removing/commenting this line here: 
https://github.com/3scale/porta/blob/master/Rakefile#L75
We are running `npm install` locally, and IIRC, we run it in our saas deployment...

Reply:
So what I mean is: we don't need to run it at all at `assets:precompile` time and not adding an extra task

Reply:
I agree the task looks a bit hacky. If we don't need the assets precompile let's remove it.

Reply:
This is for ci. If you have better idea please tell me how not to install it 3 times in circle ci?

Reply:
Idea might be using `npm ci` @didierofrivia @josemigallas ?

Reply:
What we want to achieve here? I see there are contradictory and redundant parts. We wanna have a single npm installation and a single assets compilation. I say we remove `npm i` from rake, since we have a CircleCi job that will do this and persist node_modules for next jobs.
Assets compilation might also be done during this step. This is what I run every time I wanna work with porta:
```
npm i && bundle install && bundle exec rake webpack:compile
```
And we could use `npm ci` instead, it's no magic just a slightly quicker _clean install_ that uses package-lock.